pypesto/visualize/profiles.py

```diff
@@ -6,8 +6,8 @@
 from .misc import process_result_list
 
 
-def profiles(results, fig=None, profile_indices=None, size=(18.5, 6.5),
-             reference=None, colors=None, legends=None):
+def profiles(results, ax=None, profile_indices=None, size=(18.5, 6.5),
+             reference=None, colors=None, legends=None, profile_list=0):
```

Comment:
``profile_list`` sounds like one would pass a list. ``profile_list_ix/index``?

Reply:
puh... right, that's not finished, I decided to give up on this... 😅 
The idea is to have a list of axis handles for a profiling plot, which on can pass to another call of the profiling plot... There are arbitrarily many problems with that, however... E.g. what, if you want to plot for another number of profiles than you have entries in the handle list? How to make sure that these profiles, which are plotted in the same axes, really belong to the same parameter? Etc. etc...

Reply:
if it's not used yet and maybe too difficult to implement, I'd just remove it?

Reply:
Ah, totally got your comment wrong, now I see!
Well, okay, that'll be done in 5 minutes